android/java/org/chromium/chrome/browser/crypto_wallet/util/Utils.java

```diff
@@ -1473,4 +1474,27 @@ public static Profile getProfile(boolean isIncognito) {
 
         return chromeActivity.getTabModelSelector().getModel(isIncognito).getProfile();
     }
+
+    public static org.chromium.url.internal.mojom.Origin originFromGURL(GURL address) {
+        org.chromium.url.internal.mojom.Origin hostOrigin =
+                new org.chromium.url.internal.mojom.Origin();
+        hostOrigin.scheme = address.getScheme();
+        hostOrigin.host = address.getHost();
+        hostOrigin.port = 0;
+        if (address.getPort().isEmpty()) {
+            if (hostOrigin.scheme.equals("http")) {
+                hostOrigin.port = 80;
+            } else if (hostOrigin.scheme.equals("https")) {
+                hostOrigin.port = 443;
```

Comment:
nit: can move `443` to wallet constants with label like `SECURE_TRAFFIC_PORT` OR `HTTPS_PORT` and `HTTP_PORT`

Reply:
done